CONTRIBUTING.md

```diff
@@ -58,6 +58,23 @@ $ tox -e fuzz
 $ black-primer [-k -w /tmp/black_test_repos]
 ```
 
+### News / Changelog Requirement
+
+`Black` has CI that will check for an entry corresponding to your PR in `CHANGES.md`. If
+you feel this PR not require a `CHANGES.md` please state that in a comment and a
```

Comment:
I get what you mean here, but let's be more correct

```suggestion
`Black` has CI that will check for an entry corresponding to your PR in `CHANGES.md`. If
you feel this PR not require a changelog entry please state that in a comment and a
```

ps. I'm sorry if this ticks off prettier